scripts/pyperf_env.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+set -euo pipefail
+
+apt-get update
+DEBIAN_FRONTEND=noninteractive apt-get install -y curl git build-essential iperf llvm-9-dev libclang-9-dev \
+  cmake python3 flex bison libelf-dev libz-dev
+
+cd /tmp
+
+# Install libunwind
+curl -L http://download.savannah.nongnu.org/releases/libunwind/libunwind-1.4.0.tar.gz -o libunwind-1.4.0.tar.gz
```

Comment:
Funny, this is now the 3rd libunwind we build during gProfiler's build :)

Or actually, dumb

Reply:
Not easy to solve tho

Reply:
We're also installing the same packages over and over again 🤷🏽

Reply:
yup